tracing/src/main/java/com/palantir/tracing/AsyncTracer.java

```diff
@@ -56,12 +57,21 @@ public AsyncTracer(String operation) {
         this(Optional.of(operation));
     }
 
+    public AsyncTracer(String operation, boolean createRunSpan) {
+        this(Optional.of(operation), createRunSpan);
```

Comment:
this ctor should house the full implementation, optional ctor just passes `operation.orElse(DEFAULT_OPERATION)` for backcompat